src/content/ethereum-blockchain-getting-started/index.md

```diff
@@ -0,0 +1,188 @@
+
+   ```bash
+   cd truffle-example
+   ```
+
+1. Run your migrations, specifying the network: develop
+
+   ```bash
+   truffle migrate --network develop
+   ```
+
+   This will deploy our contracts to our local blockchain. Make sure you give it the `--network develop` so it will deploy to the blockchain running on port 8545. By default it will try to deploy to port 7545 which doesn't currently have a blockchain running.
+
+   When successful, you will see the 4 example contracts deployed to your blockchain, which will cost a certain amount of ETH. Now that our contracts are deployed, we can use the Metamask browser extension to connect our blockchain to our web browser.
+
+## Install Metamask browser extension and import your accounts
+
+1. Go to https://metamask.io and install the Metamask extension for your browser of choice (Chrome or Firefox).
+
+1. Once installed, a new tab should open with the Metamask logo. Click on "Get Started" and "Create a Wallet" and agree to terms and conditions. Create a password and click "Create". You will be given a 12-word mnemonic that you can use to access or recover your Metamask wallet from any computer. Make sure to store this 12-word mnemonic in a safe and secure way. Especially if you are going to use this wallet to transact on the Ethereum Mainnet.
+
+1. Click on "Next" and it will ask you to confirm your 12-word mnemonic. If successful, it will say "Congratulations". Click on "All Done"
+
+![](./images/metamask_congratulations.png)
+
+1. Go to your Metamask and click on the top dropdown where it says "Ethereum Mainnet" to select your network and select Localhost 8545
+
+1. Click on the avatar in the top-right corner and select Import Account
+
+1. Here, you can paste one of the Private Keys that is displayed when you run `npm run ganache`. Lets copy and paste the first Private Key and click Import.
+
+1. Metamask should now display your account with the corresponding public key (displayed under Available Accounts in the Ganache CLI) with an account balance close to 100 ETH (minus the amount spent to deploy your contracts).
+
+1. If you want, you can import your other accounts, but really we only need one account right now to get started.
+
+## Start your Drizzle App
+
+1. Now that we have our Metamask and our contracts deployed to our local blockchain, lets go back to our frontend app, end the process (if still running) and restart:
+
+```bash
+npm start
+```
+
+1. We should now see our drizzle application that we can interact with our blockchain. Let's test the first contract, "Simple Storage" to make sure we can update, read, and persist data to our blockchain. Set the value to any number and submit. You should see the value update soon after.
```

Comment:
This one is showing as `1` on the page but I think should be `2`

Reply:
Thanks, good catch.